README.rst

```diff
@@ -0,0 +1,66 @@
+conda-build-prepare
+===================
+
+This repository contains a tool for unified conda package building.
+
+It extracts data from the recipe repositories and puts it into metadata of the built package.
+
+While all packages are expected to have a ``meta.yaml`` file (or, at least, a script that will prepare it), the ``conda-build-prepare`` tool populates the ``recipe_append.yaml`` file, which is automatically added to metadata (as of ``conda-build`` v3.0).
+
+Stages of preparation:
+
+#. preparing the working directory,
+#. preparing the build environment,
+#. cleaning up git tags,
+#. extracting the build environment information.
+
+Options::
+
+    package_dir         Path to the directory containing ``meta.yaml`` of the package
+    --out=DIRECTORY     Use DIRECTORY to store generated files on which ``conda build`` should be run
+    -v, --verbose       Add more verbosity to output
+
+Preparing the working directory
+-------------------------------
+
+Due to problems with Conda's own repository management [TODO: what exactly?], ``conda-build-prepare`` clones the repository by itself and allows ``conda build`` to operate on the prefetched copy.
```

Comment:
"repository management" isn't a good name here. I would say "source code git"

Reply:
I moved the line when I was updating `README.rst` with more detailed description of how it works now: https://github.com/antmicro/conda-build-prepare/pull/1/files#diff-88b99bb28683bd5b7e3a204826ead112R84

I'm not sure I understand what this first part should say. `Due to problems with Conda's own source code git` doesn't seem to be what you wanted.

Could you also give an example of those problems?